thunder/tests/test_fa3_executor.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import torch
+
+import thunder
+import thunder.core
+from thunder.executors.fa3ex import fa3_ex
+from thunder.tests.framework import requiresCUDA
+
+# flash attn 3
+try:
+    from flash_attn_interface import _flash_attn_forward, _flash_attn_backward, flash_attn_func
+
+    HAS_FA3 = True
+except:
+    HAS_FA3 = False
+
+
+@pytest.mark.parametrize("device,", ["cuda"])
+@requiresCUDA
+def test_sdpa(device: str, dtype: torch.dtype):
```

Comment:
This test isn't being executed because the test framework doesn't know how to instantiate the `dtype` parameter:

```
______________________ ERROR at setup of test_sdpa[cuda] _______________________
[gw4] linux -- Python 3.10.12 /usr/bin/python3.10
file /__w/3/s/thunder/tests/test_fa3_executor.py, line 18
  @pytest.mark.parametrize("device,", ["cuda"])
  @requiresCUDA
  def test_sdpa(device: str, dtype: torch.dtype):
E       fixture 'dtype' not found
>       available fixtures: benchmark, benchmark_weave, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, testrun_uid, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, worker_id
>       use 'pytest --fixtures [testpath]' for help on them.

/__w/3/s/thunder/tests/test_fa3_executor.py:18
```

Per @crcrpar 's point above, I think this needs to instantiate for the `float16` dtype

Reply:
My bad, I copied off https://github.com/Lightning-AI/lightning-thunder/blob/d202ba3d868642e0b4d61cd1bf93794350d5a663/thunder/tests/test_sdpaex_executor.py#L39 but removed all of https://github.com/Lightning-AI/lightning-thunder/blob/d202ba3d868642e0b4d61cd1bf93794350d5a663/thunder/tests/test_sdpaex_executor.py#L34-L36 instead of just the bfloat16 and float32 cases, fixed